simulation.py

```diff
@@ -803,10 +844,13 @@ def compute_metrics(self, time_step, **kwargs):
                 control_nodes = self.dynamics.network.get_control_nodes()
                 nodes_dict = {'treatment': treatment_nodes, 'control': control_nodes}
                 for condition in ['treatment', 'control']:
-                    self.metrics[f"avg_degree_{condition}"].append(self.dynamics.network.avg_degree(idx_list=nodes_dict[condition]))
-                    self.metrics[f"degree_variance_{condition}"].append(self.dynamics.network.degree_var(idx_list=nodes_dict[condition]))
-                    self.metrics[f"global_clustering_{condition}"].append(self.dynamics.network.global_clustering(idx_list=nodes_dict[condition]))
-                    self.metrics[f"gini_coeff_{condition}"].append(self.dynamics.network.gini_coeff(idx_list=nodes_dict[condition]))
+                    self.metrics[f"avg_degree_{condition}"].append(self.dynamics.network.avg_degree(grp=condition))
+                    self.metrics[f"global_clustering_{condition}"].append(self.dynamics.network.global_clustering(grp=condition))
+                    self.metrics[f"gini_coeff_{condition}"].append(self.dynamics.network.gini_coeff(grp=condition))
+                    if condition == 'control':
```

Comment:
also add adjusted for treatment in the case of global clustering

Reply:
done!